flumine/backtest/simulated.py

```diff
@@ -84,7 +84,14 @@ def place(
                         size,
                         runner.ex.available_to_back,
                     )
-                    return self._create_place_response(bet_id)
+                    should_expire = self._should_expire()
```

Comment:
Would this ever be True? / is it needed?

Reply:
It can be True if there is size available at the limit order's price, but not enough size to fill it.

The tests in this commit should demonstrate it: https://github.com/liampauling/flumine/pull/296/commits/178748bc0cf2d718d6506da3dcdb40366435ca06